dd-sdk-android/src/main/kotlin/com/datadog/android/log/internal/logger/LogcatLogHandler.kt

```diff
@@ -8,9 +8,12 @@ package com.datadog.android.log.internal.logger
 
 import android.os.Build
 import android.util.Log
+import com.datadog.android.Datadog
+import java.util.regex.Pattern
 
 internal class LogcatLogHandler(
-    internal val serviceName: String
+    internal val serviceName: String,
+    internal val callerNameStackIndex: Int = DEFAULT_LOGGER_CALLER_STACK_INDEX
```

Comment:
I'm not a fan of passing the absolute index in the constructor. Callers should not know about this implementation details I think. 
One solution would be to add a `nestedDepth` parameter, with default value 0 and add that to the `DEFAULT_LOGGER_CALLER_STACK_INDEX`.
Another would be to use as tag the first StackElement no in the `com.datadog.android.log.*` package ?

Reply:
I think I will go with the first one, the second one will require some extra computation which we can avoid.